challenges-webapp/src/main/webapp/vue-app/engagement-center/components/achievements/EditRealizationDrawer.vue

```diff
@@ -0,0 +1,194 @@
+<!--
+This file is part of the Meeds project (https://meeds.io/).
+Copyright (C) 2022 Meeds Association
+contact@meeds.io
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU Lesser General Public
+License as published by the Free Software Foundation; either
+version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+Lesser General Public License for more details.
+You should have received a copy of the GNU Lesser General Public License
+along with this program; if not, write to the Free Software Foundation,
+Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+-->
+<template>
+  <exo-drawer
+    ref="editRealizationDrawer"
+    class="editRealizationDrawer"
+    :right="!$vuetify.rtl"
+    @closed="close"
+    eager>
+    <template slot="title">
+      <span class="pb-2"> {{ $t('realization.label.editRealization') }} </span>
+    </template>
+    <template slot="content">
+      <v-card-text>
+        <div class="mb-2">
+          <span class="subtitle-1 labels"> {{ $t('challenges.label.reward') }}</span>
+          <v-text-field
+            v-model="points"
+            :label="$t('realization.label.points')"
+            :rules="[rules.value]"
+            :placeholder="$t('realization.label.points')"
+            class="pointsRealization"
+            type="number"
+            outlined
+            required />
+        </div>
+        <span class="subtitle-1 labels"> {{ $t('realization.label.actionLabel') }}</span>
+        <div class="actionLabel">
+          <v-textarea
```

Comment:
we may not want to use this component, please check with PO about the Testarea style to use (IT could be a CKEditor or Textare like we have in other component, see https://github.com/Meeds-io/social/blob/develop/webapp/portlet/src/main/webapp/vue-apps/profile-about-me/components/ProfileAboutMe.vue#L46-L48)